cmd/argocd/commands/cluster.go

```diff
@@ -185,6 +200,72 @@ func getRestConfig(pathOpts *clientcmd.PathOptions, ctxName string) (*rest.Confi
 	return conf, nil
 }
 
+// NewClusterEditCommand returns a new instance of an `argocd cluster edit` command
+func NewClusterEditCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
+	var (
+		clusterOptions cmdutil.ClusterOptions
+		clusterName    string
+	)
+	var command = &cobra.Command{
+		Use:   "edit cluster name",
```

Comment:
action should come after the subject.
e.g. Use: `edit SERVER/NAME`

Reply:
Done.